transaction/core/src/ring_ct/rct_bulletproofs.rs

```diff
@@ -509,11 +605,9 @@ fn sign_with_balance_check<CSPRNG: RngCore + CryptoRng, S: RingSigner + ?Sized>(
             }
         })
         .ok_or(Error::AllRingsPresigned)?;
-
```

Comment:
Why were all these new lines removed? They make it easier to read this long piece of code.

Reply:
Yeah sorry, formatting issue on my end, will fix that!